### PR TITLE
Clean up commands

### LINKED ISSUE #710: whitelist team doesn't work properly
/wl team <team name> does the same as /wl all, it doesn't properly test for the right team

to reproduce:
join blue team, make sure you are not on the whitelist already
type /wl team red
you will be added to the whitelist while being on blue

tested in: 4f2c109

### PR DESCRIPTION
**Pull request by dentmaged**
This PR adds more commands and cleans up commands. This hopefully closes #710.

### CODE REVIEW

**Comment by CaptainElliott** [comment]
:+1:

File: src/main/java/in/twizmwaz/cardinal/command/PunishmentCommands.java [hunk 1]
```diff
@@ -57,8 +57,8 @@ public static void warn(CommandContext cmd, CommandSender sender) throws Command
         }
     }
 
-    @Command(aliases = {"ban", "pb"}, usage = "<player> [reason]", desc = "Ban a player.", min = 1)
-    @CommandPermissions("cardinal.punish.ban")
+    @Command(aliases = {"ban", "permaban", "pb"}, usage = "<player> [reason]", desc = "Ban a player.", min = 1)
+    @CommandPermissions("cardinal.punish.permaban")
```
Comment (Pablete1234) [inline src/main/java/in/twizmwaz/cardinal/command/PunishmentCommands.java]: I don't see a reason to change the permission unless you are adding a /tempban <player> <time> [reason]

Comment (CaptainElliott) [inline src/main/java/in/twizmwaz/cardinal/command/PunishmentCommands.java]: There is already `/pb` which is short for a permanent ban, and the ban is permanent so it makes sense

**Comment by dentmaged** [comment]
Commit amended to allow more flexibility on each languages.

File: src/main/java/in/twizmwaz/cardinal/command/ModeratorCommand.java [hunk 2]
```diff
@@ -1,22 +1,28 @@
 package in.twizmwaz.cardinal.command;
 
-import com.sk89q.minecraft.util.commands.Command;
-import com.sk89q.minecraft.util.commands.CommandContext;
-import com.sk89q.minecraft.util.commands.CommandException;
-import com.sk89q.minecraft.util.commands.CommandPermissions;
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/command/ModeratorCommand.java]: Why are we moving these imports?

File: src/main/java/in/twizmwaz/cardinal/Cardinal.java [hunk 3]
```diff
@@ -114,6 +112,9 @@ public boolean hasPermission(CommandSender sender, String perm) {
         cmdRegister.register(ScoreCommand.class);
         cmdRegister.register(ProximityCommand.class);
         cmdRegister.register(BroadcastCommands.class);
+
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/Cardinal.java]: random blank line

File: src/main/java/in/twizmwaz/cardinal/command/ProximityCommand.java [hunk 4]
```diff
@@ -56,4 +56,5 @@ public static void proximity(final CommandContext cmd, CommandSender sender) thr
             }
         } else throw new CommandPermissionsException();
     }
+
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/command/ProximityCommand.java]: random blank line

File: src/main/java/in/twizmwaz/cardinal/command/PrivateMessageCommands.java [hunk 5]
```diff
@@ -57,4 +58,5 @@ public static void reply(final CommandContext cmd, CommandSender sender) throws
             sender.sendMessage(new LocalizedChatMessage(ChatConstant.ERROR_PLAYER_DISABLED_PMS, target.getDisplayName()).getMessage(((Player) sender).getLocale()));
         }
     }
+
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/command/PrivateMessageCommands.java]: random blank line

File: src/main/java/in/twizmwaz/cardinal/command/StatsCommand.java [hunk 6]
```diff
@@ -24,4 +24,5 @@ public static void cardinal(final CommandContext cmd, CommandSender sender) {
             sender.sendMessage(ChatColor.GRAY + " KD: " + ChatColor.GOLD + (Math.round(GameHandler.getGameHandler().getMatch().getModules().getModule(Stats.class).getKdByPlayer(player) * 100.0) / 100.0));
         }
     }
+
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/command/StatsCommand.java]: random blank line

File: src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java [hunk 7]
```diff
@@ -243,25 +178,21 @@ public static OfflinePlayer matchSinglePlayer(CommandSender sender, String rawUs
             // look up player according to the who is online now
             List<Player> players = Bukkit.getServer().matchPlayer(rawUsername);
             switch (players.size()) {
-                case 0:
-                    throw new CommandException("No players matched query. Use @<name> for offline lookup.");
-                case 1:
-                    return players.get(0);
-                default:
-                    throw new CommandException("More than one player found! Use @<name> for exact matching.");
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: Did your IDE change this or did you? I feel like the indentation made sense here

**Comment by twizmwazin** [comment]
Check those line notes, and also this may have issues when rebased on top of the teams branch.

File: src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java (hunk 7 above)
Comment (dentmaged) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: @twizmwazin Yeah, sorry!

File: src/main/java/in/twizmwaz/cardinal/Cardinal.java (hunk 3 above)
Comment (dentmaged) [inline src/main/java/in/twizmwaz/cardinal/Cardinal.java]: @twizmwazin That makes it neater (in my opinion).

File: src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java (hunk 7 above)
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java]: You can leave this if your IDE changed it

**Comment by TiTo-418** [comment]
@dentmaged This PR should be rebased

**Comment by dentmaged** [comment]
@TiTo-418 I no longer contribute to Cardinal as I have my own PGM plugin to work on.

**Pull request closed without merging**